api/envoy/api/v2/route/route.proto

```diff
@@ -437,6 +439,27 @@ message RouteAction {
     google.protobuf.Duration per_try_timeout = 3 [(gogoproto.stdduration) = true];
   }
 
+  // Specifies the idle timeout for the route. If not specified, this defaults
```

Comment:
How does one fully disable it if they want? Do we tell them to set a really big value? Or should we support 0 to mean no timeout? Thoughts?